Graphs/TreeGraph/index.js

```diff
@@ -287,7 +286,19 @@ const TreeGraph = (props) => {
         const links = treeData.descendants().slice(1);
 
         // Normalize for fixed-depth.
-        nodes.forEach(d => { d.y = d.depth * 250 });
+        let depth, count = 0;
+        nodes.forEach(d => {
```

Comment:
why have you chosen these specific numbers: 250, 40 and 60 below? What happens on bigger/smaller resolutions?